wcfsetup/install/files/lib/system/database/table/index/DatabaseTableIndex.class.php

```diff
@@ -169,7 +169,7 @@ public function type(?string $type): self
     /**
      * Returns a `DatabaseTableIndex` object with the given name.
      */
-    public static function create(string $name): self
+    public static function create(string $name): static
```

Comment:
This change is incorrect: The method actually returns `self`, not `static`. Doesn't matter, due to the final, but it should be consistent.

I recommend to also change the `return` to `return new static($name)`.

Reply:
I changed the `return` to `return new static($name)`. Now return type `static` should be correct.